samples/subsys/smf/smf_calculator/prj.conf

```diff
@@ -0,0 +1,37 @@
+CONFIG_DEBUG=y
+CONFIG_LOG=y
+CONFIG_SHELL=y
+
+# Needed for boards that enable RTT backends for logging
+# e.g. nrf52840dk/nrf52840 and any others that enable it
+CONFIG_LOG_BACKEND_RTT=n
+
+# Enable the state machine framework
+CONFIG_SMF=y
+CONFIG_SMF_ANCESTOR_SUPPORT=y
+CONFIG_SMF_INITIAL_TRANSITION=y
+
+# Enable floating point support
+CONFIG_PICOLIBC_IO_FLOAT=y
+CONFIG_CBPRINTF_FP_SUPPORT=y
+
+# Enable thread awareness for debugging tools supporting it
+CONFIG_DEBUG_THREAD_INFO=y
+
+# enable to use thread names
+CONFIG_THREAD_NAME=y
+
+# Display Options
+CONFIG_DISPLAY=y
+CONFIG_DISPLAY_LOG_LEVEL_ERR=y
+
+# LVGL Options
+CONFIG_LVGL=y
+CONFIG_LV_Z_MEM_POOL_SIZE=16384
+CONFIG_LV_Z_SHELL=y
+# Percentage of screen size for a video buffer
+# 14% ~= 32KB RAM on adafruit_2_8_tft_touch_v2
+CONFIG_LV_Z_VDB_SIZE=14
```

Comment:
I would let the board's default options be used there, unless there is a good reason not to?

Reply:
There's a default option for LVGL that is more video ram than my little `disco_l475_iot1` board has in total. There doesn't seem to be board-specific LVGL overrides.

I can add more explanation.